drupal/rootfs/etc/islandora/utilities.sh

```diff
@@ -284,6 +284,43 @@ function fedora_url {
     fi
 }
 
+
+# Allow modifications to settings.php by changing ownership and perms
+function allow_settings_modifications {
+    local site="${1}"; shift
+    local drupal_root=$(drush drupal:directory)
+    local subdir=$(drupal_site_env "${site}" "SUBDIR")
+    local site_directory=$(realpath "${drupal_root}/sites/${subdir}")
+
+    # send debug output to stderr because the caller typically captures output from this function.
+    #>&2 echo "adjusting ownership of ${site_directory}/settings.php"  
+    if [ -f "${site_directory}/settings.php" ]; then
+        previous_owner_group=$(stat -c "%u:%g" "${site_directory}/settings.php")
```

Comment:
On review I don't know why we'd ever have an owner/group other than nginx:nginx (100:101)? With the possible exception if it's bind mounted. Possibly all files in `/var/www/drupal` have the hosts users group to allow for host edits or something... That being said it doesn't hurt anything. Just a thought as to why did we do this in the first place.

Reply:
Good catch re: permissions.  I've moved this into the function.

Restoring ownership is only important when it's bind mounted.  It ensures the file remains readable/writable by the host user.  I've added a comment in the code to this effect.